taxonomy/views.py

```diff
@@ -0,0 +1,79 @@
+"""
+Views for the taxonomy app
+"""
+
+from django.shortcuts import render, redirect
+
+from .forms import VocabularyForm
+from learningresources.models import Repository
+from taxonomy.models import Vocabulary
+
+
+def edit(request, vocabulary_id=None):
```

Comment:
It should definitely be at a different URL, but it should use the same template or at least shared template fragment. It can be a different view, and it's so small it may as well be to avoid some `if` statements in the edit view to handle both.